root/src/main/java/com/example/servingwebcontent/ConnectDatabase.java

```diff
@@ -15,31 +15,66 @@ public ConnectDatabase (String database, String user, String pass) {
 		}
 	}
 
-	public ResultSet queryDB (String query) {
+	public ResultSet getMovie (String id) {
 		ResultSet rs = null;
+		String query = "select * from movies where id=?";
 		try {
-			Statement stmt = con.createStatement();
-			rs = stmt.executeQuery(query);
+			PreparedStatement stmt = con.prepareStatement(query);
+			stmt.setString(1, id);
+			rs = stmt.executeQuery();
 		} catch (SQLException e) { 
-			throw new RuntimeException("failed to query db" ,e); 
+			throw new RuntimeException("failed to query db", e); 
+		}
+
+		return rs;
+	}
+
+	public ResultSet updateMovie (String id, String column, String value) {
```

Comment:
What if the user wants to update multiple attributes of a movie at once? This is only supporting a single column at a time. We should probably just take in a movie object as input.